<!--
  Copyright (C) 2018-2021 LEIDOS.
  Licensed under the Apache License, Version 2.0 (the "License"); you may not
  use this file except in compliance with the License. You may obtain a copy of
  the License at
  http://www.apache.org/licenses/LICENSE-2.0
  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
  License for the specific language governing permissions and limitations under
  the License.
-->
<!--
  This file is used to launch a ros wrapper of Autoware planner as a CARMA strategic plugin.
-->
<launch>
    <rosparam command="load" file="$(find bsm_generator)/config/parameters.yaml"/>
    <node name="bsm_generator" pkg="bsm_generator" type="bsm_generator">
      <remap from="gnss_fix_fused" to="$(optenv CARMA_INTR_NS)/gnss_fix_fused"/>
      <remap from="imu_raw" to="$(optenv CARMA_INTR_NS)/imu_raw"/>
      <remap from="velocity_accel_cov" to="$(optenv CARMA_INTR_NS)/velocity_accel_cov"/>
      <remap from="ekf_twist" to="$(optenv CARMA_INTR_NS)/vehicle/twist"/>
    </node>
</launch>
